slick.grid.js

```diff
@@ -1533,6 +1535,12 @@ if (typeof Slick === "undefined") {
       columnsById = {};
       for (var i = 0; i < cols.length; i++) {
         var m = cols[i];
+
+        if (m.json && m.json.isHidden) {
+          m.width = 0;

```

Comment:
Is width=0 really all that's needed to prevent rendering? Could I have 500 hidden columns with no draw performance impact?